react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import './HotButton.css';
+
+export default function HotButton({ text, color }) {
+  // we are creating a state called clicks and initializing it to null with setClicks as the setter.
```

Comment:
Since you are calculating the color below, it doesn't make much sense to have it as a prop. You also don't pass in a color. It would be best to just remove it and make `color` a local variable.